<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-U3S0M86" failures="0" tests="4" name="test.HomePageTest" time="34.437" errors="1" timestamp="2023-02-03T20:22:34 IST" skipped="3">
  <testcase classname="test.HomePageTest" name="init" time="34.436">
    <error message="null value in entry: url=null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: null value in entry: url=null
at com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:33)
at com.google.common.collect.SingletonImmutableBiMap.<init>(SingletonImmutableBiMap.java:43)
at com.google.common.collect.ImmutableBiMap.of(ImmutableBiMap.java:81)
at com.google.common.collect.ImmutableMap.of(ImmutableMap.java:128)
at org.openqa.selenium.remote.DriverCommand.GET(DriverCommand.java:191)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:296)
at testbase.TestBase.initialization(TestBase.java:60)
at test.HomePageTest.init(HomePageTest.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:823)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:590)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- init -->
  <system-out/>
  <testcase classname="test.HomePageTest" name="validateUserBtn" time="0.001">
    <skipped/>
  </testcase> <!-- validateUserBtn -->
  <system-out/>
  <testcase classname="test.HomePageTest" name="validateCorporateBtn" time="0.000">
    <skipped/>
  </testcase> <!-- validateCorporateBtn -->
  <system-out/>
  <testcase classname="test.HomePageTest" name="validateLogoutTest" time="0.000">
    <skipped/>
  </testcase> <!-- validateLogoutTest -->
  <system-out/>
</testsuite> <!-- test.HomePageTest -->
